tests/ballerina-integration-test/src/test/resources/websub/services/websub_publisher.bal

```diff
@@ -0,0 +1,157 @@
+import ballerina/http;
```

Comment:
Shall we add copyright headers to bal files as well?

Reply:
Fixed in all places.